atomic_reactor/source.py

```diff
@@ -145,7 +145,8 @@ def commit_id(self):
         return self.lg.commit_id
 
     def get(self):
-        return self.lg.git_path
+        self.source_path = self.lg.git_path
+        return self.source_path
```

Comment:
removed setting it. I tried to ensure it's equal to set path but then realized there's almost no way of changing it midway and even if it was changed it would clone into the same directory as self.source_path.